extensions/amp-story-panning-media/0.1/test/test-amp-story-panning-media.js

```diff
@@ -121,5 +125,39 @@ describes.realWin(
         }px)`
       );
     });
+
+    it('calculates transform x with lock-bounds', async () => {
+      const attributes = {
+        'group-id': 'group-1',
+        'x': '50%',
+        'zoom': '2',
+        'lock-bounds': '',
+      };
+      await createAmpStoryPanningMedia(
+        '/examples/amp-story/img/conservatory-coords.jpg',
+        attributes
+      );
+      await storeService.dispatch(Action.CHANGE_PAGE, {id: 'page1', index: 0});
+      await panningMedia.layoutCallback();
+      await afterRenderPromise();
+      expect(panningMedia.animateTo_.x).to.equal(panningMedia.maxBounds_.left);
```

Comment:
Is there a way we can get these values without using private properties? We try to avoid using these in our tests, since they could be changed/deleted and are not meant to be used directly in tests.